docs/contributing/coding_guide.rst

```diff
@@ -450,31 +446,59 @@ Imports
 * Do not use star imports (e.g., :py:`from package.subpackage import *`),
   except in very limited situations.
 
-Requirements
-------------
+Project infrastructure
+======================
 
-* Package requirements are specified in |pyproject.toml|_. |tox.ini|_
-  also contains a testing environment for the minimal dependencies.
+.. role:: toml(code)
+   :language: TOML
```

Comment:
📝 This directive defines the `:toml:` role, which lets us get fancy color formatting from pygments 🌈 for in-line code.  It's similar to a `.. code-block:: toml` directive.

😅 I thought I dreamed of a new color, but it was a pigment of my imagination.